src/app/(mobile-ui)/layout.tsx

```diff
@@ -132,34 +139,33 @@ const Layout = ({ children }: { children: React.ReactNode }) => {
                 </div>
             )}
             <div
-                className={classNames('flex w-full flex-1 overflow-x-visible overflow-y-scroll', {
+                className={classNames('z-1 flex w-full flex-1 overflow-x-visible overflow-y-scroll', {
                     'p-4': !isHome,
                 })}
-                style={{
-                    backgroundColor: walletColor,
-                }}
             >
-                {showWaitlistScreen ? <HomeWaitlist /> : children}
+                {peanutWalletIsInPreview ? <HomeWaitlist /> : children}
             </div>
-            <div className="grid grid-cols-5 border-t-2 border-black p-2">
-                {tabs.map((tab) => {
-                    if (tab.icon === 'wallet') {
-                        return <WalletToggleButton />
-                    }
+            {!peanutWalletIsInPreview && (
+                <div className="grid grid-cols-5 border-t-2 border-black p-2">
+                    {tabs.map((tab) => {
+                        if (tab.icon === 'wallet') {
+                            return <WalletToggleButton />
+                        }
 
-                    return (
-                        <Link
-                            href={tab.href}
-                            key={tab.name}
-                            className={classNames('flex flex-row justify-center py-2 hover:cursor-pointer ', {
-                                'text-purple-1': pathName === tab.href,
-                            })}
-                        >
-                            <NavIcons name={tab.icon} size={30} />
-                        </Link>
-                    )
-                })}
-            </div>
+                        return (
+                            <Link
```

Comment:
These are bottom tabs for the mobile UI they're not used anywhere else, seems fine as is no ? Can refactor the component yes for sure, but tailwind will stay

Reply:
Generally if we have the same code repeated in 4+ places it just makes sense to standardize

This is non-blocking but it would've been cleaner if we did this